Dockerfile

```diff
@@ -0,0 +1,25 @@
+# syntax=docker/dockerfile:1
+FROM --platform=$BUILDPLATFORM golang:1.17 as build
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+
+WORKDIR /go/src/dnsproxy/
+
+COPY . .
+
+RUN set -e \
+    && echo "Running on $BUILDPLATFORM, building for $TARGETPLATFORM" \
+    && apt-get update \
+    && apt-get install --no-install-recommends -y ruby \
+    && ruby docker-env.rb
```

Comment:
Size: add `rm -rf /var/lib/apt/lists/`

Reply:
It's `builder` part, no mind of sizing, dns-proxy runing on busybox for more arch supporting.

Reply:
Sure, you're right. I didn't realize it, just force of habit. :smile: